tutorials/dataframe/distrdf003_live_visualization.py

```diff
@@ -0,0 +1,70 @@
+## \file
+## \ingroup tutorial_dataframe
+## \notebook -draw
+## Configure a Dask connection and visualize the filling of a 1D and 2D
+## histograms distributedly.
+##
+## This tutorial showcases the process of setting up real-time data representation 
+## for distributed computations.
+## By calling the LiveVisualize function, you can observe the canvas updating
+## with the intermediate results of the histograms as the
+## distributed computation progresses. 
+##
+## \macro_code
+## \macro_image
+##
+## \date August 2023
+## \author Silia Taider
+from dask.distributed import LocalCluster, Client
+import ROOT
+
+# Import the live visualization fucntion
+from DistRDF import LiveVisualize
```

Comment:
Also in tutorials, hide the `DistRDF` name and just show the usage as a submodule of ROOT
```suggestion
LiveVisualize = ROOT.RDF.Experimental.Distributed.LiveVisualize
```